Dabfile

```diff
@@ -0,0 +1,24 @@
+build() {
```

Comment:
Everything from this point to the next comment was copied verbatim from meke.sh, excluding the extra indentation

Reply:
ni